support-frontend/assets/pages/supporter-plus-landing/setup/threeTierChecks.ts

```diff
@@ -31,9 +30,7 @@ export const threeTierCheckoutEnabled = (
 	if (isWeeklyCheckout) {
 		const urlParams = new URLSearchParams(window.location.search);
 		return (
-			urlParams.get(
-				`threeTierCreateSupporterPlusSubscription${isTierThreeV2 ? 'V2' : ''}`,
-			) === 'true'
+			urlParams.get('threeTierCreateSupporterPlusSubscriptionV2') === 'true'
```

Comment:
is this an instance of using "V2" we should keep?

Reply:
Thought best to do a 2nd seperate PR to uncouple this part, its directly related to unpicking this [PR](https://github.com/guardian/support-frontend/pull/6125) - a V2 path taken through a State Machine.